### ISSUE TITLE
Add option to export scan results as "markdown"

### ISSUE THREAD

**Issue by bluetechy** [nexB/scancode-toolkit#3017]
## Short Description

As a user, I would like a command line option (e.g. `--markdown`) so that I can export the scan results in "markdown" format.  This would enable the user to upload these markdown pages to a github wiki.

## Possible Labels

- new feature

## Select Category

-  [x] Enhancement
-  [ ] Add License/Copyright
-  [ ] Scan Feature
-  [ ] Packaging
-  [ ] Documentation
-  [ ] Expand Support
-  [ ] Other

## **Describe the Update**

At the moment, there is already support for exporting scan results to JSON, CSV, HTML, and to a customize template.  Being that markdown is one of the most popular formats used by programmers to document things in their repositories and wikis, it would be nice to have a quick way of doing this rather than have to create a custom template.  I have tried converting HTML to markdown with a few different python libraries and it never converts properly mainly because the usage of the `<caption>` tag in the HTML format and it also leaves CSS styling in the page.

## **How This Feature will help you/your organization**

This would allow me to commit the scan results to a wiki page (which renders markdown), for example, to github's wikis.

## **Possible Solution/Implementation Details**

1. Add a new command line option such as `--markdown FILE` that will format the scan results accordingly; or,
2. Modify the HTML formatting so that it can better converted by (python) libraries like `markdownify` or `html2markdown`.

## **Example/Links if Any**

```
docker run -v $PWD/:/output scancode-toolkit -clpeui --processes 4 --markdown /output/$FILENAME /output
```

**Comment by bluetechy** [comment]
I would prefer that the markdown resembles more like the HTML format than the CSV format.  I am able to convert from CSV format to MD format, but previewing it is ugly and hard to read....that is why I prefer the HTML style of formatting for the markdown rather than the CSV style of formatting.

**Comment by pombredanne** [comment]
@bluetechy Thanks for the report. You wrote:

> [...]
> it would be nice to have a quick way of doing this rather than have to create a custom template. 
> [...]
> This would allow me to commit the scan results to a wiki page (which renders markdown), for example, to github's wikis.

Have you tried to craft a custom template for this? These are standard Jinja2 templates and  a markdown output could be as simple as an alias to a "custom" output. 

This template work would have to done first in all cases. Do you mind to take a stab at it?

See https://scancode-toolkit.readthedocs.io/en/stable/rst_snippets/custom_output_format.html?highlight=jinja for some hints.
